content/en/docs/File Format/implementationstatus.md

```diff
@@ -13,94 +13,96 @@ implementations.
 The value in each box means:
 * ✅: supported
 * ❌: not supported
+* (R/W): partial reader/writer only support
```

Comment:
Added an extra piece in legend to allow partial reader- or writer-only support. Happy to remove it and leave the corresponding boxes blank if needed